components/EditTagsModal.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import { gql, useMutation, useQuery } from '@apollo/client';
+import { Form, Formik } from 'formik';
+import { FormattedMessage } from 'react-intl';
+
+import { IGNORED_TAGS } from '../lib/constants/collectives';
+import { API_V2_CONTEXT, gqlV1 } from '../lib/graphql/helpers';
+import { Collective } from '../lib/graphql/types/v2/graphql';
+
+import CollectiveTagsInput from './CollectiveTagsInput';
+import { Flex } from './Grid';
+import MessageBox from './MessageBox';
+import StyledButton from './StyledButton';
+import StyledInputFormikField from './StyledInputFormikField';
+import StyledModal, { ModalBody, ModalFooter, ModalHeader } from './StyledModal';
+import { TOAST_TYPE, useToasts } from './ToastProvider';
+
+const editTagsMutation = gqlV1`
+  mutation editCollective(
+    $collective: CollectiveInputType!
+  ) {
+    editCollective(
+      collective: $collective
+    ) {
+      id
+      tags
+    }
+  }
+`;
```

Comment:
Is there a GraphQL V2 mutation we could use to achieve that?

Reply:
There is `editAccount` in V2 but it does not currently have tags editing implemented.

Reply:
Would be great to support this in v2 indeed.

Reply:
Doesn't have to be in this PR, but would be great to add it to `editAccount` or to use the new `setTags` mutation introduced in https://github.com/opencollective/opencollective-api/pull/8558